vagrant/provisioning/roles/pki/tasks/main.yml

```diff
@@ -133,7 +133,7 @@
     type: RSA
     size: 2048
     mode: 0644
-    cipher: "AES-256-ECB"
+    cipher: "auto"
```

Comment:
What's with auto vs AES-256-ECB? The specific cipher is better, I believe. What if auto changes the cipher? Perhaps, a better/upgraded cipher would be used, but can this lead to issues like using a cipher that is not supported by some tool using the key?

Reply:
Since we are changing pyOpenSSL with Cryptography the only option we can use here is auto since the installer will fail

Note from Ansible module overview page: https://docs.ansible.com/ansible/latest/collections/community/crypto/openssl_privatekey_module.html
**When using the cryptography backend, use auto.**